airbyte-integrations/connectors/source-vitally/pyproject.toml

```diff
@@ -3,26 +3,25 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "0.1.1"
+version = "0.1.2"
 name = "source-vitally"
-description = "Source implementation for Vitally."
-authors = [ "Elliot Trabac <elliot.trabac1@gmail.com>",]
+description = "Source implementation for vitally."
+authors = [ "Airbyte <contact@airbyte.io>",]
 license = "MIT"
 readme = "README.md"
 documentation = "https://docs.airbyte.com/integrations/sources/vitally"
 homepage = "https://airbyte.com"
 repository = "https://github.com/airbytehq/airbyte"
-[[tool.poetry.packages]]
-include = "source_vitally"
+packages = [ { include = "source_vitally" }, {include = "main.py" } ]
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "1.0.0"
+airbyte-cdk = "^0"
```

Comment:
I'm curious. Is there a reason we shouldn't be using `1.0.0`?

Reply:
nope actually it was from generator :sweat_smile: